src/App.tsx

```diff
@@ -1,17 +1,78 @@
 import { useTranslation } from 'react-i18next'
+import LangSwitcher from './components/LangSwitcher/LangSwitcher'
+import logo from './assets/logo.svg'
+import gifAdult from './assets/bg-animation.gif'
+import gifCh from './assets/bg-animation-ch.gif'
+import flowers from './assets/flowers.svg'
+import flowersMobile from './assets/flowers-2.svg'
+import './scss/sections/hero.scss'
 
 function App() {
-	const { t } = useTranslation()
+	const { t, i18n } = useTranslation()
 
 	return (
 		<>
-			<h1>Vite + React</h1>
-			<div className="card">
-				<h2 className="t__h2">{t('heroTitle')}</h2>
-			</div>
-			<div>
-				<p>ПОРУЧ ДІТИ</p>
-			</div>
+			<section className="hero">
+				{i18n.language === 'ch' && (
+					<img
+						src={window.innerWidth <= 375 ? flowersMobile : flowers}
+						className={window.innerWidth <= 375 ? 'hero__bg-flowers-mobile' : 'hero__bg-flowers'}
+					/>
+				)}
+				<img
+					src={i18n.language === 'ad' ? gifAdult : gifCh}
+					alt="Background animation"
+					className={i18n.language === 'ad' ? 'hero__bg-animation' : 'hero__bg-animation-ch'}
+				/>
+				<div className="hero__container">
+					<div className="hero__switch-and-menu-container">
+						<div className="hero__lang-switch-container">
+							<p className="t__text-thin">ПОРУЧ ДІТИ</p>
+							<LangSwitcher />
+						</div>
+						<div className="hero__menu-button-container">
+							<p className="t__text-thin">МЕНЮ</p>
+							<svg
+								className="hero__menu-button"
+								width="81"
+								height="80"
+								viewBox="0 0 81 80"
+								fill="none"
+								xmlns="http://www.w3.org/2000/svg"
+							>
+								<circle cx="40.7021" cy="40" r="39" stroke="#39C8D3" strokeWidth="2" />
+								<path
+									d="M21.7021 35H69.7021"
+									stroke="#39C8D3"
+									strokeWidth="2"
+									strokeLinecap="round"
+								/>
+								<path
+									d="M12.7021 45H60.7021"
+									stroke="#39C8D3"
+									strokeWidth="2"
+									strokeLinecap="round"
+								/>
+							</svg>
+						</div>
+					</div>
+					<div>
+						<picture>
+							<img src={logo} alt="Main logo" className="hero__logo" />
+						</picture>
+					</div>
+					<div className="hero__text-block-container">
+						<div className="hero__text-block">
```

Comment:
Why is there a div here if it is already in the

Reply:
It's there to align the text block to flex end